### PR TITLE
Add FindName Command

### PR DESCRIPTION
**Pull request by ytingtan**
Find command currently returns persons whose details and tags contain keywords. Very general search function. Users might want to search for specific attributes of a person, in this base by name only.

Let's,

    Create more specific find commands based on specific attributes of a person, in this case, based on name

#16 #19 #31

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindNameCommand.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof FindCommand // instanceof handles nulls
+                && predicate.equals(((FindCommand) other).predicate)); // state check
+    }
+}
```
Comment (emptygx) [inline src/main/java/seedu/address/logic/commands/FindNameCommand.java:42]: Missing new line at EOF.

**Review by emptygx — DISMISSED** [review]
Screenshot 2022-10-07 at 11 33 12 AM
Errors that need to be fixed to pass the CI!

File: src/main/java/seedu/address/logic/commands/FindNameCommand.java (hunk 1 above)
Comment (emptygx) [reply src/main/java/seedu/address/logic/commands/FindNameCommand.java]: The new line should be after the last curly brace! Sorry should have been clearer 😢

Comment (ytingtan) [reply src/main/java/seedu/address/logic/commands/FindNameCommand.java]: Got it, thank you! 🙏

**Review by Josephaedan — APPROVED** [review]
LGTM, test cases passed

**Pull request merged**